src/twisted/web/http_headers.py

```diff
@@ -208,20 +209,35 @@ def setRawHeaders(self, name, values):
         @param values: A list of strings each one being a header value of
             the given name.
 
+        @raise TypeError: Raised if C{values} is not a L{list} of L{bytes}
+            or L{unicode} strings, or if C{name} is not a L{bytes} or
+            L{unicode} string.
```

Comment:
Should we still be documenting things as `unicode` in this brave new Python 3 world? I was tempted to change these in my last PR.

Reply:
We should switch to `str`.  There is no ambiguity left there.